spec/models/miq_widget/content_generator_spec.rb

```diff
@@ -1,8 +1,9 @@
-describe MiqWidget::ContentGenerator do
+RSpec.describe MiqWidget::ContentGenerator do
   let(:content_generator) { described_class.new }
 
   describe "#generate" do
-    let(:widget) { double("MiqWidget", :id => 123, :title => "title", :log_prefix => "") }
+    let(:widget) { instance_double("MiqWidget", :id => 123, :title => "title", :log_prefix => "") }
+    let(:records) { instance_double(ActiveRecord::Relation) }
```

Comment:
Almost feels like just tearing out all the doubles and using the db instead would be simpler / less fragile.